### DISCUSSION TITLE
last update doens't generate docs in notebooks cells

### DISCUSSION THREAD

**Discussion by matrs** [community/community#66621] [Copilot Conversations]
### Select Topic Area

Bug

### Body

```
Version: 1.82.0
Commit: 8b617bd08fd9e3fc94d14adb8d358b56e3f72314
Date: 2023-09-06T22:07:18.759Z
Electron: 25.8.0
ElectronBuildId: 23503258
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Linux x64 5.15.128-1-MANJARO
```
`github copilot v1.110.389`
`jupyter v2023.8.1002501831`

Since the last update, in jupyter notebooks, copilot is unable to generate docs for functions. It doesn't matter if the cell only contains one function definition or multiple ones and if one uses the command  palette or the right click  menu. Also, if I only select the name of the function, doesn't change anything. In the previous version (august), this problem wasn't present. For python files (`.py`), generating docs works as expected.

![image](https://github.com/community/community/assets/12502882/af2088b8-30cd-4bca-a7d7-475b42b82454)

**Comment by Akash1134** [comment]
Hi @matrs ,

Please troubleshoot Copilot in JetBrains using this [documentation](https://docs.github.com/en/copilot/troubleshooting-github-copilot/viewing-logs-for-github-copilot-in-your-environment?tool=jetbrains). 

Thanks!

**Reply by matrs** [reply]
@Akash1134,  I'm using vscode (did you see the specs i posted at the beggining of my post?!) , not jetbrains.
